README.md

```diff
@@ -57,7 +61,11 @@ For use with AWS CDK v1:
 pip install datadog-cdk-constructs
 ```
 
-## Go Package Installation:
+##### Note:
+
+Pay attention to the output from your package manager as the `Datadog CDK Construct Library` has peer dependencies.
```

Comment:
Moving it up to the `PyPI` section, where it's supposed to be, as shown in the original PR https://github.com/DataDog/datadog-cdk-constructs/pull/28.